### PR TITLE
feat: add chunking_strategy kwarg

### PR DESCRIPTION
**Pull request by Coniferish**
Add `chunking_strategy` as a kwarg for parsing and associated test.

### Testing
```
curl -X 'POST' 'http://127.0.0.1:8000/general/v0/general' \
 -H 'accept: application/json'  \
 -H 'Content-Type: multipart/form-data' \
 -F 'files=@sample-docs/layout-parser-paper-fast.pdf' \
 -F 'chunking_strategy=by_title' \
 | jq -C . | less -R
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,6 +187,29 @@ curl -X 'POST'
  | jq -C . | less -R
 ```
 
+
+#### Chunking Elements
+
+Set the `chunking_strategy` to chunk text into larger or smaller elements. Defaults to `None` with optional arg of `by_title`.
```
Comment (awalker4) [inline README.md:193]: Can you clarify in here that these params are defaults and not exposed yet?

**Review by awalker4 — COMMENTED** [review]
Looks good! Can you start a new version in the CHANGELOG? You'll need to run `make version-sync` once that's updated.

File: README.md (hunk 1 above)
Comment (Coniferish) [reply README.md:193]: Sorry, but what do you mean by "not exposed" yet?

Comment (Coniferish) [reply README.md:193]: Ohh... do you mean for `multipage_sections` and the other additional params?

Comment (awalker4) [reply README.md:193]: Yeah, perhaps a note that they'll be in the api soon, and here are the defaults for now.

Comment (Coniferish) [reply README.md:193]: Should I go ahead and add them for this PR?

Comment (awalker4) [reply README.md:193]: Your call! Might as well if they're all similar.

**Review by awalker4 — APPROVED** [review]
LGTM!

**Pull request merged**